spec/controllers/v2/tailorings_controller_spec.rb

```diff
@@ -253,6 +247,32 @@
       end
     end
 
+    context 'with authorization only via cert_auth' do
+      before do
+        header = JSON.parse(Base64.decode64(current_user.account.identity_header.raw))
+        header['identity']['auth_type'] = Insights::Api::Common::IdentityHeader::CERT_AUTH
+        request.headers['X-RH-IDENTITY'] = Base64.encode64(header.to_json)
+      end
```

Comment:
Simply setting up the context won't work, there's logic in the authentication concern ensuring a passthrough over RBAC is cert_auth is valid for some endpoints. All these endpoints are defined for V1, in V2 we have different ones. If his test is passing, there are bigger issues as well...